README.md

```diff
@@ -252,6 +254,26 @@ If the user is authenticated then your API route will simply execute, but if the
 
 All cookies will be set as `HttpOnly` cookies and will be forced to HTTPS (`Secure`) if the application is running with `NODE_ENV=production` and not running on localhost.
 
+## Troubleshooting
+
+### Error `id_token issued in the future, now 1570650460, iat 1570650461`
+
+Increase the clock tolerance for id_token validation:
+
+```js
+import { initAuth0 } from '@auth0/nextjs-auth0';
+
+export default initAuth0({
+  ...
+  session: {
+    ...
+  },
+  oidcClient: {
+    clockTolerance: 10 // In seconds.
```

Comment:
Shouldn’t this be in milliseconds?

Reply:
According to [their docs](https://auth0.com/docs/libraries/auth0js/v9), it is in seconds.


> leeway (Integer) A value in seconds; leeway to allow for clock skew with regard to JWT expiration times.

Reply:
Correct, should be milliseconds to have a consistent way to express timespan (eg: on httpTimeout it's also MS).

Reply:
I updated the readme, thanks for letting me know.